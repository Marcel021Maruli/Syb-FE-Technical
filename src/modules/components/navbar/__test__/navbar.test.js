import { render } from "enzyme";
import { BrowserRouter } from "react-router-dom";
import Navbar from "../";

describe("<Navbar />", () => {
  const wrapper = render(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>,
  );
  it("should match with a snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should have default className", () => {
    expect(wrapper.hasClass("Navbar")).toBeTruthy();
  });
});
