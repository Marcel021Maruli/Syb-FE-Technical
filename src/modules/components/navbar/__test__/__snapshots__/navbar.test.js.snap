// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should match with a snapshot 1`] = `
<div
  class="Navbar"
>
  <div
    class="ant-row Navbar-ul"
    style="margin-left:-16px;margin-right:-16px"
  >
    <div
      class="ant-col ant-col-xs-4 ant-col-sm-4 ant-col-md-2"
      style="padding-left:16px;padding-right:16px"
    >
      <a
        class="Navbar-link--inactive"
        href="/anilist-list"
      >
        <img
          alt="Anilist Logo"
          class="Navbar-logo"
          src="anilist-logo.png"
        />
      </a>
    </div>
    <div
      class="ant-col ant-col-xs-16 ant-col-sm-16 ant-col-md-8"
      style="padding-left:16px;padding-right:16px"
    >
      <input
        class="Search"
        name="search"
        placeholder="Search Title.."
        type="text"
      />
    </div>
  </div>
</div>
`;
